flexget/plugins/output/qbittorrent.py

```diff
@@ -0,0 +1,83 @@
+from __future__ import unicode_literals, division, absolute_import
+
+import logging
+
+from requests import Session
+
+from flexget import plugin
+from flexget.event import event
+
+log = logging.getLogger('qbittorrent')
+
+
+class OutputQBitTorrent(object):
+    """
+    Example:
+
+      qbittorrent:
+        username: <USERNAME> (default: (none))
+        password: <PASSWORD> (default: (none))
+        host: <HOSTNAME> (default: localhost)
+        port: <PORT> (default: 8080)
+        movedone: <OUTPUT_DIR> (default: (none))
+        label: <LABEL> (default: (none))
+    """
+    schema = {
+        'type': 'object',
+        'properties': {
+            'username': {'type': 'string'},
+            'password': {'type': 'string'},
+            'host': {'type': 'string'},
+            'port': {'type': 'integer'},
+            'movedone': {'type': 'string'},
+            'label': {'type': 'string'}
+        }
```

Comment:
I think we probably need a couple more things here, additionalProperties False for one, and if any of the options are required, that should also be specified.